libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -723,13 +723,13 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 		case FunctionType::Kind::Revert:
 		{
 			if (arguments.empty())
-				m_context.appendRevert();
+				m_context.appendRevert("Revert without reason");
```

Comment:
This is not an internal revert, so we should not modify the message of the user.